docs/docsite/keyword_desc.yml

```diff
@@ -27,7 +27,7 @@ delay: Number of seconds to delay between retries. This setting is only used in
 delegate_facts: Boolean that allows you to apply facts to a delegated host instead of inventory_hostname.
 delegate_to: Host to execute task instead of the target (inventory_hostname). Connection vars from the delegated host will also be used for the task.
 diff: "Toggle to make tasks return 'diff' information or not."
-environment: A dictionary that gets converted into environment vars to be provided for the task upon execution. This cannot affect Ansible itself nor its configuration, it just sets the variables for the code responsible for executing the task.
+environment: A dictionary that gets converted into environment vars to be provided for the task upon execution. This cannot affect any plugins aside from modules (no lookups, callbacks, etc) nor Ansible itself nor its configuration, it just sets the variables for the code responsible for executing the task.
```

Comment:
```suggestion
environment: A dictionary that gets converted into environment vars to be provided for the task upon execution. This can ONLY be used with modules. This isn't supported for any other type of plugins nor Ansible itself nor its configuration, it just sets the variables for the code responsible for executing the task.
```